sql_metadb/derived_tables/agreements_package_content_item.sql

```diff
@@ -254,4 +212,86 @@ CREATE INDEX ON agreements_package_content_item (res_publication_type_label);
+
+COMMENT ON COLUMN agreements_package_content_item.ti_first_author IS 'For monographs (books), the last name of the book’s first author';
+
+COMMENT ON COLUMN agreements_package_content_item.ti_monograph_edition IS 'For monographs (books), edition of the book';
+
+COMMENT ON COLUMN agreements_package_content_item.ti_monograph_volume IS 'For monographs (books), the volume number of the book';
+
+COMMENT ON COLUMN agreements_package_content_item.ti_first_editor IS 'For monographs (books), the last name of the book’s first editor';
+
+COMMENT ON COLUMN agreements_package_content_item.identifier_id IS 'UUID of identifier';
+
+COMMENT ON COLUMN agreements_package_content_item.identifier_value IS 'Value of the identifier (i.e. the identifier string)';
+
+COMMENT ON COLUMN agreements_package_content_item.identifier_namespace_id IS 'UUID of identifier_namespace';
+
+COMMENT ON COLUMN agreements_package_content_item.identifiernamespace_name IS 'Value of the namespace (i.e. the namespace name as a string)';
+
+COMMENT ON COLUMN agreements_package_content_item.entitlement_id IS 'UUID of Entitlement (aka Agreement Line)';
+
+COMMENT ON COLUMN agreements_package_content_item.res_name IS 'Name of the resource (can be name of a package or other resource type or the title of a published work such as a book or journal)';
+
+COMMENT ON COLUMN agreements_package_content_item.res_sub_type_fk IS 'ID of reference data value for the subtype of resource';
+
+COMMENT ON COLUMN agreements_package_content_item.res_sub_type_value IS 'A string value which is used to identify the refdata value';
+
+COMMENT ON COLUMN agreements_package_content_item.res_sub_type_label IS 'A string label for the rdv_value';
+
+COMMENT ON COLUMN agreements_package_content_item.res_sub_type_category IS 'Text string identifying the refdata category';
+
+COMMENT ON COLUMN agreements_package_content_item.res_type_fk IS 'ID of reference data value for the type of resource';
+
+COMMENT ON COLUMN agreements_package_content_item.res_type_value IS 'A string value which is used to identify the refdata value';
+
+COMMENT ON COLUMN agreements_package_content_item.res_type_label IS 'A string label for the rdv_value';
+
+COMMENT ON COLUMN agreements_package_content_item.res_type_category IS 'Text string identifying the refdata category';
+
+COMMENT ON COLUMN agreements_package_content_item.res_publication_type_fk IS 'ID of the publication type';
+
+COMMENT ON COLUMN agreements_package_content_item.res_publication_type_value IS 'A string value which is used to identify the refdata value';
+
+COMMENT ON COLUMN agreements_package_content_item.res_publication_type_label IS 'A string label for the rdv_value';
+
+COMMENT ON COLUMN agreements_package_content_item.res_publication_type_category IS 'Text string identifying the refdata category';
```

Comment:
This has the same comment as res_type_category. Maybe embellish the comments so users can tell the difference between the two values?